app/models/partners/profile.rb

```diff
@@ -123,7 +121,7 @@ def client_share_total
 
     def check_social_media
       return if website.present? || twitter.present? || facebook.present? || instagram.present?
-      return if partner&.partials_to_show&.exclude?("media_information")
+      return if partner.partials_to_show.exclude?("media_information")
```

Comment:
Does this open us up to more 500s?

Reply:
Maybe, but this is an undo of what we had a week ago.